docs/developers/get_started/getting_started_android_cmake.md

```diff
@@ -140,7 +140,7 @@ Translate a source MLIR into IREE module:
 
 ```shell
 # Assuming in IREE source root
-$ ../iree-build-host/install/bin/iree-translate \
+$ ../iree-build-host/install/bin/iree-compile \
   -iree-mlir-to-vm-bytecode-module \
```

Comment:
(future cleanup?)
Can we now drop the explicit `-iree-mlir-to-vm-bytecode-module` from our docs when using `iree-compile`?